examples/siw-jquery/src/main/java/com/okta/spring/example/ExampleApplication.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2017 Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.spring.example;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+
+@SpringBootApplication
+@EnableGlobalMethodSecurity(prePostEnabled = true)
+public class ExampleApplication {
+
+//    @Bean
+//    protected GlobalMethodSecurityConfiguration methodSecurityConfiguration() {
+//        return new GlobalMethodSecurityConfiguration() {
+//            @Override
+//            protected MethodSecurityExpressionHandler createExpressionHandler() {
+//                return new OAuth2MethodSecurityExpressionHandler();
+//            }
+//        };
+//    }
```

Comment:
I'd remove rather than comment. If you're trying to show another option, add a comment to the block that says so.